z2/resources/start_node.tera.sh

```diff
@@ -8,17 +8,19 @@ NODE_PRIVATE_KEY=""
 CONFIG_FILE="${CHAIN_NAME}.toml"
 CHECKPOINT_FILE=""  # The optional checkpoint file to mount, if provided.
 DATA_FOLDER=$(pwd)/data
+SCILLA_SERVER_PORT=3111
 
 # Define a function to show help using EOF
 help() {
     cat <<EOF
         Usage: ${0} -k <node-private-key> [-c <config-file>] [-p <checkpoint-file>]
 
         Options:
-            -k, --key           NODE_PRIVATE_KEY (mandatory)
-            -c, --config        Path to the config file (optional, default: ${CHAIN_NAME}.toml)
-            -p, --checkpoint    Path to the checkpoint file (optional, it is needed only the first time a node is started)
-            -h, --help          Display this help message
+            -k, --key                   NODE_PRIVATE_KEY (mandatory)
+            -c, --config                Path to the config file (optional, default: ${CHAIN_NAME}.toml)
+            -p, --checkpoint            Path to the checkpoint file (optional, it is needed only the first time a node is started)
+                --scilla-server-port    Specify Scilla Server port
```

Comment:
Should we mention here that `consensus.scilla_address` needs to match?

Also if we're updating the default, we should update the default address in `cfg.rs` too.